tests/unittest/test_algo.py

```diff
@@ -37,6 +37,8 @@ def config(self):
         algo2 = Algo2()
 
         s = DummyStreamer()
+        # Prevent streamer from running which will cause an infinite loop
+        s.start = lambda: None
```

Comment:
prevent the dummy streamer from running `start` and going into an infinite loop.